icons/traffic-lights.svg

```diff
@@ -0,0 +1,21 @@
+<svg
```

Comment:
Not sure about the design of this one. On first sight for me I didn't recognized it as a traffic light.